packages/ui/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChevronDownIcon } from '@heroicons/react/24/solid';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { Badge } from './Badge';
+
+const Icon = ChevronDownIcon;
+
+export default {
+    title: 'Buttons/Badge',
```

Comment:
I wonder, if this component is under `Button` section, acts like a button, and has all of the same layout and styles, why can't this just be a prop of the `Button` component? (e.g. `rounded`).

If it walks like a duck and quacks like a duck.. 😉

Reply:
Will make this a prop of the button